### DISCUSSION TITLE
qtile bar not fit my screen

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#2969] [Q&A] [answered]
I did edit the config.py bar fit on my screen no problem
and i installed linux mint for new user linux guide type video ı'm done 
I installed arch linux but this time it doesn't fit my screen I don't know why?

sorry my english not good

![2021-10-31-092354_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/139576074-1a52225a-e44b-4444-8276-5d7a541d2ab2.png)

**Comment by elParaguayo** [comment, marked as answer]
It looks like you've got too much in your bar.

Qtile will not shrink widgets' text size etc to make sure everything fits.

You've got a lot of decorations in your bar which are very large. You may want to make them smaller.

**Reply by ghost** [reply]
there was no problem before

**Reply by elParaguayo** [reply]
"before" what?

Could be margins, font, font size etc.

Do you have a screenshot from "before" i.e. when it did work?

**Reply by ghost** [reply]
thanks again ı change settings again good day bro
